package.json

```diff
@@ -2,10 +2,10 @@
   "name": "ackee-frontend",
   "displayName": "Ackee frontend",
   "description": "A set of usefull snippets and commands we use at Ackee for web apps development.",
-  "version": "0.1.0",
+  "version": "0.1.2",
```

Comment:
Where has version 0.1.1 lost? 😄

Reply:
I accidentaly run the publish script twice 😅